src/instructlab/configuration.py

```diff
@@ -364,6 +364,12 @@ class _evaluate(BaseModel):
     gpus: Optional[int] = Field(
         default=None, description="Number of GPUs to use for running evaluation."
     )
+    format: Optional[str] = Field(
+        default="text",
+        description="Format of evaluation results. Defaults to 'text' which prints to stdout with additional logging, also allows for 'json' to print to stdout without additional logging",
```

Comment:
What's the additional logging in this description? The writing out to a log file? From this description it seems like `text` will also have logging to a file, and `json` won't. Is that what we intend to do?